test/mocha.opts

```diff
@@ -1,3 +1,3 @@
+--compilers ts:ts-node/register
 --recursive
 --reporter dot
-dist/test
```

Comment:
Please revert changes in this file, they are breaking the setup introduced by #2.